apps/frontend/src/app/api/mutations/mutateCriteria.ts

```diff
@@ -0,0 +1,15 @@
+import { ICriteria } from '@innbyggerpanelet/api-interfaces';
+import { mutate } from 'swr';
+import { poster } from '../operations';
+
+export const createCriteria = async (criteria: ICriteria) => {
```

Comment:
I think the file name should match the content of the file :)

Reply:
I agree, but was thinking about adding PATCH and PUT functionality in the same file in the future. Mutation being the general term. Would you rather we split them? :)

Reply:
If the intention is to add more mutation features later it is ok